### ISSUE TITLE
Island Reset whilst on Cooldown

### ISSUE THREAD

**Issue by BartBarends** [illyria-io/SkyblockX#68]
When you reset your island when you are on cooldown, it will delete your old island, but not make a new one. You don't have an island for the remainder of the cooldown

![image](https://user-images.githubusercontent.com/55838233/86783024-18947f80-c060-11ea-8cb0-0d7902ef7d6d.png)

**Comment by prosavage** [comment]
Yeah so what do you suggest we do, not allow deletion until he can create?

**Comment by BartBarends** [comment]
Yes, only delete if he can make a new island
